tests/testthat/cassettes/ropenaq-encoding.yaml

```diff
@@ -0,0 +1,869 @@
+http_interactions:
```

Comment:
a bit big for sure but I don't know which part provoked the failure.

Reply:
that's okay